Resources/translations/SonataArticleBundle.en.xliff

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="article">
+                <source>article</source>
+                <target>Article</target>
+            </trans-unit>
+            <trans-unit id="fragment">
+                <source>fragment</source>
+                <target>Fragment</target>
+            </trans-unit>
+            <trans-unit id="close">
+                <source>close</source>
+                <target>Close</target>
+            </trans-unit>
+            <trans-unit id="delete">
+                <source>delete</source>
+                <target>Delete</target>
+            </trans-unit>
+            <trans-unit id="fragment._default.label_plus">
+                <source>fragment._default.label_plus</source>
+                <target>Add new text</target>
+            </trans-unit>
+            <trans-unit id="fragment.media.label_plus">
+                <source>fragment.media.label_plus</source>
+                <target>Add new media</target>
+            </trans-unit>
+            <trans-unit id="article.versions">
+                <source>article.versions</source>
+                <target>Article versions</target>
+            </trans-unit>
+            <trans-unit id="status">
+                <source>status</source>
+                <target>Status</target>
+            </trans-unit>
+            <trans-unit id="version">
+                <source>version</source>
+                <target>Version</target>
+            </trans-unit>
+            <trans-unit id="last.update.date">
+                <source>last.update.date</source>
+                <target>Last update date</target>
+            </trans-unit>
+            <trans-unit id="publication.date.start">
```

Comment:
Use dots to indicate hierachy : `publication.date_range.start` or `publication.start_date`